laws/ord.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const {precedes, equals} = require('..');
+
+/**
+   ### Ord
+
+   1. `a.precedes(a) === true` (reflexivity)
+   2. If `a.precedes(b)` and `b.precedes(a)`, then `a.equals(b)` (antisymmetry)
+   3. If `a.precedes(b)` and `b.precedes(c)`, then `a.precedes(c)` (transitivity)
+
+**/
+
+const reflexivity = t => eq => x => {
+  const y = t(x);
+
+  const a = y[precedes](y);
+  const b = true;
+  return eq(a, b);
+};
+
+const antisymmetry = t => eq => x => {
+  const f = t(x);
+  const g = t(x);
+
+  const a = f[precedes](g);
+  const b = g[precedes](f);
+  const c = g[equals](f);
+  const d = true;
+  return eq(a, d) && eq(b, d) && eq(c, d);
+};
+
+const transitivity = t => eq => x => {
```

Comment:
I defined `precedes` on `Id` which may have been wrong as it should just delegate to `value`'s `precedes`. Should I define it in `internal/func.js` instead? I could implement it for `String` and `Number` and delegate otherwise.